### ISSUE TITLE
[Issue]: Actor Thumbnails don't show up until you click on the actor for each movie.

### ISSUE THREAD

**Issue by gummykage** [jellyfin/jellyfin#8447]
### Please describe your bug

1.) Using docker in container station of QNAP NAS do a pull on jellyfin/jellyfin and select latest image.
2.) Setup directories for Movies / Actors / expose /dev/dri/ in Enviroment PATH=/dev/dri/ to enable hardware acceleration.
3.) Create library point to media folder.
4.) Click on Library after scan and select Movie to see actors on bottom.
5.) Actors won't load thumbnails. 
Checking the cache folder shows that actor thumbnails are not made until you click on them.

![image](https://user-images.githubusercontent.com/9695713/192076334-5aa73e62-0e3b-44ec-ac5e-aa0978a64d9a.png)

Clicking on actor shows the image is downloaded correctly:
![image](https://user-images.githubusercontent.com/9695713/192076363-dc3bb504-4992-4f50-b0c1-4291922dfe96.png)

Reloading previous page now shows the thumbnail. And the thumbnail is confirmed to be in the cache directory of actors.
![image](https://user-images.githubusercontent.com/9695713/192076383-a3c9d324-529e-4860-ac25-25fc60d17e40.png)

### Jellyfin Version

Other

### if other:

10.8.4

### Environment

```markdown
- OS:QNAP Unbuntu
- Virtualization:Docker
- Clients:Browser, Jellyfin Media Player
- Browser: Chrome and Brave Latest
- FFmpeg Version: 5.1
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:NAT
- Storage:SSD
```

### Jellyfin logs

```shell
[2022-09-24 02:30:00.570 +00:00] [WRN] [37] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://192.168.0.50:8097/Users/b86b55ff29b940e2a2f194dd7466880e/Items/180aa308f7177e57a6c7e30d246d5d8b" to "192.168.0.2" in 0:00:02.2100474 with Status Code 200
[2022-09-24 02:30:01.210 +00:00] [WRN] [39] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://192.168.0.50:8097/Items/180aa308f7177e57a6c7e30d246d5d8b/Images/Primary?fillHeight=810&fillWidth=540&quality=96&tag=297449c16ea947f1803706d3ec814db3" to "192.168.0.2" in 0:00:00.6137169 with Status Code 200
[2022-09-24 02:30:11.214 +00:00] [INF] [22] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.0.2" closed
[2022-09-24 02:30:11.929 +00:00] [INF] [37] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.0.2" request
[2022-09-24 02:30:12.966 +00:00] [WRN] [33] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://192.168.0.50:8097/Items/de56f312b31fc74ea638a936d64360a0/Images/Primary?fillHeight=810&fillWidth=540&quality=96&tag=c23e1b81a1d99b3e17f447a5cc85997b" to "192.168.0.2" in 0:00:00.6461174 with Status Code 200
[2022-09-24 02:30:13.852 +00:00] [WRN] [37] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://192.168.0.50:8097/Items/de56f312b31fc74ea638a936d64360a0/Images/Backdrop/0?maxWidth=1920&tag=591d76694be8f1380c4c5d168aa37e5a&quality=80" to "192.168.0.2" in 0:00:01.5568075 with Status Code 200
[2022-09-24 02:32:57.280 +00:00] [INF] [23] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.0.2" closed
[2022-09-24 02:33:06.418 +00:00] [INF] [35] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.0.2" request
[2022-09-24 02:33:07.454 +00:00] [WRN] [41] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://192.168.0.50:8097/Items/de56f312b31fc74ea638a936d64360a0/Images/Backdrop?fillHeight=219&fillWidth=389&quality=96&tag=591d76694be8f1380c4c5d168aa37e5a" to "192.168.0.2" in 0:00:00.6477422 with Status Code 200
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gummykage** [comment]
Reverting to image docker jellyfin/jellyfin 10.8.0 the issue is not longer present.
![image](https://user-images.githubusercontent.com/9695713/192077069-96aac9ef-53d4-4ad0-bd89-5c42743c8f69.png)

**Comment by gummykage** [comment]
Another note this is not happening on 10.8.4 installed on Windows 10 Enterprise (fully updated). So this may be something going on only with 10.8.4 docker image. I will test 10.8.3 next.

![image](https://user-images.githubusercontent.com/9695713/192077349-b28e3b92-d12f-45ee-bfcc-2f7e0cd6781f.png)

**Comment by gummykage** [comment]
Appears this was fixed on 10.8.4-1 and the Unstable build. It's working after downloading and trying those two images in docker.
